session.go

```diff
@@ -594,6 +564,7 @@ func (s *session) ExecutePreparedStmt(stmtID uint32, args ...interface{}) (ast.R
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	PrepareTxnCtx(s)
 	st := executor.CompileExecutePreparedStmt(s, stmtID, args...)
```

Comment:
`executor.GetInfoSchema(s)` can be used in the function of `CompileExecutePreparedStmt`.